index.html

```diff
@@ -4,10 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">   
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="header"></div>
+    <div class="header">
+        <img src="/image/images.png" width="100px">
```

Comment:
- [ ] Good job working on setup and mobile-first 👍 , however, as the previous reviewer said, wrap the "Headline" section in a `section` tag. It represents a generic standalone section of the document, so would be nice to use the `section` tag. Kindly make sure to follow this structure 😃 

`````bash
<body>
    <main>
        <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
        </header>
        <section>
               --- the headline section (right after the header) ----
        </section>
    </main>
</body>
`````